collections/sentinel-1-grd.yaml

```diff
@@ -163,7 +163,7 @@ Summaries:
     - HV
     - VV
     - VH
-  sar:bands:
+  raster:bands:
     - name: VV
       description: 'Single co-polarization, vertical transmit/vertical receive'
       polarization: VV
```

Comment:
I will check with Matthias but would suggest something like this for `openeo:gsd`:

```
openeo:gsd:
  value:
  - - 10
    - 10
  - - 25
    - 25
  - - 40
    - 40
  unit: m
```

which would look in the json as this

```
"openeo:gsd": {
    "value": [
        [
            10,
            10
        ],
        [
            25,
            25
        ],
        [
            40,
            40
        ]
    ],
    "unit": "m"
}
```

Reply:
That would nicely solve it. Thanks for checking with him!

Reply:
Got confirmation on that (https://github.com/openEOPlatform/architecture-docs/issues/128#issuecomment-1021227060). Let's add this for all bands + remove polarization from the raster:bands as mentioned by Matthias as it's already present in `sar:polarizations` above.

Reply:
That is now done. 

Can you also review https://github.com/eurodatacube/public-collections/pull/161 on which this MR is based on and to which this MR will be merged? Thanks!

Reply:
Changed to degrees as well

`10m => 1/3" = 0.3333333333" = 0.00009259259°` (using `1/3" = 10 m`)
`25m => 1/27" * 25 = 0.037037037" * 25 = 0.925925926" = 0.0002572016°` (using `1/27" = 1 m`)
`40m => 1/3" * 4 = 1.333333333" =  0.0003703704°` (using `1/3" = 10 m`)

rules taken from https://www.usna.edu/Users/oceano/pguth/md_help/html/approx_equivalents.htm

Reply:
See comment https://github.com/eurodatacube/public-collections/pull/163#discussion_r793538063